ggu/main.c

```diff
@@ -331,7 +331,7 @@ process_single_packet(struct rte_mbuf *pkt, const struct ggu_config *ggu_conf)
 /* Information needed to submit GGU packets to the GGU block. */
 struct ggu_request {
 	/* GT-GK Unit packets. */
-	struct rte_mbuf *pkts[GATEKEEPER_MAX_PKT_BURST];
+	struct rte_mbuf **pkts;
```

Comment:
Make `pkts` the last field of `struct ggu_request`, so we can maintain the single `malloc()`.

Reply:
Notice that here it's critical to have a single `malloc()` because `struct ggu_request` is allocated from a poll of memory reserved for this struct, whereas a general `malloc()` is a lot slower. This is critical for the hot paths of Gatekeeper.

Reply:
Do we really need to allocate additional memory here? Isn't it enough to make it point to the array (i.e., ``struct rte_mbuf **pkts``)?

Reply:
Wouldn't callers need to allocate this array with `malloc()`?

Reply:
Yes, I think the array should be allocated using ``malloc()`` here, as we can only allocate fixed-size entries from mailbox.